### PR TITLE
Feature submit on enter

### PR DESCRIPTION
**Pull request by adisen**
### What does this pull request aim to achieve? 💯
To allow login and signup form to be submitted on "Enter"

_Please be concise and link any related issue(s):_
Issue [787](https://github.com/FightPandemics/FightPandemics/issues/787)

### CODE REVIEW

**Comment by adisen** [comment]
> Thanks @adisen .
> 
> Functionality works but we need keep the existing design for disabled buttons, at least `background-color: rgb(66, 90, 242);` is the main difference I see.
> 
> Also there's some other unintended affect of changing from using button from "antd-mobile" to "antd" button here e.g (left is this PR, right is staging):
> 
> image
> 
> 
> 
> 
> 
> 

I'll look into it. "antd-mobile" doesn't support the kind of button needed and I also feared some of the other Buttons have broken. Fixed the one I saw.

File: client/src/components/Button/BaseButton.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ const BaseButton = styled(Button)`
       pointer-events: none;
     `}
 
+
```
Comment (idjevm) [inline client/src/components/Button/BaseButton.js:70]: remove this extra line

Comment (adisen) [reply client/src/components/Button/BaseButton.js:70]: Okay

Comment (idjevm) [reply client/src/components/Button/BaseButton.js:69]: oh nooo, now this is merged into master. It's fine though. but please remove extra white lines next time.

**Comment by idjevm** [comment]
@joshmorel I think commits like this one, it will be really better to have squash and merged. We have 7 commits that are irrelevant to the actual change that was made.

**Comment by adisen** [comment]
Okay @idjevm. Noted

**Pull request merged**